### DISCUSSION TITLE
unresolved import in latest RC

### DISCUSSION THREAD

**Discussion by deathbreakfast** [rwf2/Rocket#1936] [Questions]
I'm trying to use `rocket::response::content::RawHtml` and for some reason I'm getting an unresolved import when the cargo dependency version is set to "0.5.0-rc".

I'm able to repo this with the provided examples. Here is a before and after snippet of examples/cookies/Cargo.toml
```
[package]
name = "cookies"
version = "0.0.0"
workspace = "../"
edition = "2018"
publish = false

[dependencies]
rocket = { path = "../../core/lib", features = ["secrets"] }

[dependencies.rocket_dyn_templates]
path = "../../contrib/dyn_templates"
features = ["handlebars"]
```

is changed to this

```
[package]
name = "cookies"
version = "0.0.0"
workspace = "../"
edition = "2018"
publish = false

[dependencies]
rocket = { version = "0.5.0-rc", features = ["secrets"] }

[dependencies.rocket_dyn_templates]
path = "../../contrib/dyn_templates"
features = ["handlebars"]

```

I am new to rust, not sure if I'm missing any thing fundamental with dependencies or if I can get some general direction would be really helpful.

**Comment by jebrosen** [comment]
In 0.5, this type was renamed from `RawHtml` to [`rocket::response::content::Html`](https://api.rocket.rs/v0.5-rc/rocket/response/content/struct.Html.html).
